lib/atc/gcp/storage_uploader.rb

```diff
@@ -34,11 +34,15 @@ def upload_file(local_file_path, object_key, **options)
                                       calculate_crc32c(local_file_path, verbose: options[:verbose])
 
     puts 'Performing upload...' if options[:verbose]
-    bucket.create_file(
-      local_file_path, object_key,
-      content_type: BestType.mime_type.for_file_name(local_file_path),
-      crc32c: precalculated_whole_file_crc32c, metadata: options[:metadata]
-    )
+
+    Retriable.retriable(on: [Google::Cloud::UnavailableError], tries: 3) do
```

Comment:
Good thought.  From the few times that we've run into this, parallel requests running a couple of seconds later were usually okay (though parallel requests running within the same second have had issues).  The way that the retry works is that it tries, waits 0.5 seconds, tries again, waits `0.5 * 1.5` (0.75) seconds, tries again, and then waits `0.5 * 1.5 * 1.5` (1.125) seconds.  Maybe for good measure, I'll increase the base 0.5 to 1, so that way it'll try 1 second, then 1.5 seconds, then 2.25 seconds.  And if we still run into the issue, I'll increase it further.  Thanks!